packages/ckeditor5-dev-env/lib/index.js

```diff
@@ -26,26 +26,57 @@ const tasks = {
 	 * @param {String} options.cwd Current working directory (packages) from which all paths will be resolved.
 	 * @param {String} options.packages Where to look for other packages (dependencies).
 	 * @param {Array.<String>} options.skipPackages Name of packages which will be skipped.
+	 * @param {Boolean} options.isDevPackage Whether the changelog will be generated for development packages.
+	 * @param {Boolean} [options.checkPackageJson=true] If set to false, the mechanism will not check whether
+	 * the current package being specified in 'package.json' file.
 	 * @returns {Promise}
 	 */
 	generateChangelogForDependencies( options ) {
 		const execOptions = {
 			cwd: options.cwd,
 			packages: options.packages,
-			skipPackages: options.skipPackages || []
+			skipPackages: options.skipPackages || [],
```

Comment:
I think that since this is a config option and not a property of some stateful object, it's fine that it's called like the `--skip` option. Besides, it may even become a `{Function|Array}` one day and what's then?